src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -1049,4 +1084,30 @@ export default class MutationTable<
             />
         );
     }
+
+    protected getMutations() {
+        let data: Mutation[][] | undefined = [];
+        if (this.props.dataStore) {
+            data = this.props.dataStore.allData;
+        } else if (this.props.data) {
+            data = this.props.data;
+        }
+        return data;
+    }
+
+    protected hasRequiredASCNProperty(property: string): boolean {
```

Comment:
For all the places using this method, you did a not logic before. Should you just return the opposite logic?`doesNotHaveRequiredASCNProperty`?

Reply:
might be a personal preference but I prefer the way it is currently written instead of reversing the logic